js/cards.js

```diff
@@ -0,0 +1,188 @@
+    live: 'https://ruthmy.github.io/portfolio/',
+    github: 'https://github.com/ruthmy',
+  },
+  {
+    image: 'images/snapshoot-portfolio.png',
+    imagesrc: 'images/img-plaholder.png',
+    imagealt: '',
+    title: ['Multi-Post Stories Gain+Glory', 'Keeping track of hundreds of components 6'],
+    languages: ['Ruby on rails', 'css', 'JavaScript', 'html', 'Codekit', 'GitHub', 'JavaScript', 'Bootstrap', 'Terminal', 'Codepen'],
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    live: 'https://ruthmy.github.io/portfolio/',
+    github: 'https://github.com/ruthmy',
+  },
+];
+
+/// //////This section is to create the cards \\\\\\\\\\\\\\\\\\\\
+
+// Get the body element so you can show the overlay on the hole screen
+const body = document.querySelector('body');
+
+// This variable is for the cards, I'm put one by one in here and then it will be sent.
+let insideCard = '';
+
+indexCardsArray.forEach((singleCardContents) => {
+  // This line let me see that I'm actually get it the array
+  // and let me see each of the object rendered in the console.
+  // console.log(singleCardContent);
+
+  // Here, I put teh cards one by one in the previous empty insideCard variable.
+  // This way I create the hole HTML syntax for the Recent Works section
+  insideCard
+        += `
+        <section class="recent-works__card">    
+            <div class="recent-works__img">
+                <img src="${singleCardContents.imagesrc}" alt="${singleCardContents.imagealt}">
+                <div div class="recent-works__info">
+                    <h3 class="recent-works__title">${singleCardContents.title[0]}</h3>
+                    <ul class="recent-works__tags">
+                        <li class="recent-works__tag">${singleCardContents.languages[0]}</li>
+                        <li class="recent-works__tag">${singleCardContents.languages[1]}</li>
+                        <li class="recent-works__tag">${singleCardContents.languages[2]}</li>
+                        <li class="recent-works__tag">${singleCardContents.languages[3]}</li>
+                    </ul>
+                    <button class="recent-works__link" href="#">See Project</button>
+                </div>
+            </div>
+        </section>
+        `;
+  // uncomment the line below to see the output in the browser console.
+  // console.log(insideCard);
```

Comment:
- [ ] As this is simulating a review for a production-level code, kindly remove the `console` comments, as your reviewer is at the discretion to use a professional way to debug your code, which means your responsibility is to keep your code as clean as possible. such information can be added to your README under `usage` if you want to 👍